extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -159,6 +159,17 @@ export class DraggableDrawer extends AMP.BaseElement {
       this.containerEl_.insertBefore(spacerEl, this.contentEl_);
       this.contentEl_.appendChild(headerShadowRootEl);
       this.element.classList.add('amp-story-page-attachment-ui-v2');
+      this.headerEl_.classList.add('amp-story-page-attachment-ui-v2');
+      const handler = (entries) => {
+        this.headerEl_.classList.toggle(
+          'i-amphtml-story-draggable-drawer-header-stuck',
+          !entries[0].isIntersecting
+        );
+      };
+      const observer = new window.IntersectionObserver(handler);
```

Comment:
Nit: It would be more apparent when reading the code if the handler was either declared inside the observer or called `observerHandler`